stop_speed_epsilon: 1.0
cmv_id: CARMA_TEST_CMV
cargo_id: TEST_CARGO
host_id: HOST_ID
